inc/admin/class-storefront-plugin-install.php

```diff
@@ -110,7 +110,7 @@ public static function install_plugin_button( $plugin_slug, $plugin_file, $plugi
 				?>
 				<span class="plugin-card-<?php echo esc_attr( $plugin_slug ); ?>">
 					<a href="<?php echo esc_url( $button['url'] ); ?>" class="<?php echo esc_attr( $button['classes'] ); ?>" data-originaltext="<?php echo esc_attr( $button['message'] ); ?>" data-name="<?php echo esc_attr( $plugin_name ); ?>" data-slug="<?php echo esc_attr( $plugin_slug ); ?>" aria-label="<?php echo esc_attr( $button['message'] ); ?>"><?php echo esc_attr( $button['message'] ); ?></a>
-				</span> or
+				</span> <?php esc_html_e( 'or', 'storefront' ); ?>
```

Comment:
This needs context since "or" alone could mean many things. I'd suggest using https://developer.wordpress.org/reference/functions/_x/ and wrapping that in `echo esc_html(` to maintain the escaping also.

Reply:
Agreed, this is not ideal in its current state – I'll take a look and get it tidied up.